python/servo/build_commands.py

```diff
@@ -69,8 +67,8 @@ def get_rustc_llvm_version() -> Optional[List[int]]:
             if line_lowercase.startswith("llvm version:"):
                 llvm_version = line_lowercase.strip("llvm version:")
                 llvm_version = llvm_version.strip()
-                version = llvm_version.split(".")
-                print(f"Info: rustc is using LLVM version {'.'.join(version)}")
+                version = list(map(int, llvm_version.split(".")))
+                print(f"Info: rustc is using LLVM version {llvm_version}")
```

Comment:
This doesn't look equivalent to the old cold, which would join the various parts of the version with a '.'.